src/services/find-available-port.service.js

```diff
@@ -22,7 +22,7 @@ export default () =>
       // Similar command to lsof
       // Finds if the specified port is in use
       const command = /^win/.test(os.platform())
-        ? `netstat -aon | find "${port}"`
+        ? `netstat -aon | C:\\WINDOWS\\system32\\find "${port}"`
         : `lsof -i :${port}`;
       childProcess.exec(command, (err, res) => {
```

Comment:
Do you think it would be cleaner if we used `cwd` option?

```javascript
childProcess.exec(command, {
  cwd: isWin() ? 'C:\\Windows\\Sytem32' : null
}, (err, res) => { (err, res) => { 
  /* ... */
});
```